app/components/threads_button/threads_button.tsx

```diff
@@ -113,10 +111,6 @@ const ThreadsButton = ({
         return [container, icon, text, badge];
     }, [customStyles, isActive, onCenterBg, styles, unreads, isOnHome]);
 
-    if (groupUnreadsSeparately && (onlyUnreads && !isActive && !unreads && !mentions)) {
```

Comment:
Why this change?

Reply:
basically, this condition only works when `groupUnreadsSeparately` is true, and the expected behavior when `groupUnreadsSeparately` is true is we want the threads button to be always visible